dashboard/src/features/transactions/components/New/FormActionItem.jsx

```diff
@@ -58,13 +58,13 @@ export default class ActionItem extends React.Component {
 
   render() {
     const {
+      id,
       type,
       account_id,
       account_alias,
       control_program,
       transaction_id,
       position,
-      output_id,
```

Comment:
This affects this file and dashboard/src/features/transactions/components/New/New.jsx only.